tests/console/validate_repos.pm

```diff
@@ -13,8 +13,11 @@ use base "consoletest";
 use testapi;
 use repo_tools 'validate_repo_properties';
 use scheduler 'get_test_suite_data';
+use version_utils qw(is_agama);
 
 sub run {
+    return if (is_agama && check_var('FLAVOR', 'Full'));
```

Comment:
I don't follow this, why we don't want to validate repos for full medium? we should.

Reply:
Isn't it that the FULL medium hnas everything it nees on the FULL medium? If I look at `sles_default` test for the FULL medium we don't do a product registration... so where shall the repos come from?

Reply:
I heard from Chenzi that we don't need check repos of Full on SLE16. Please @chcao provide bug id or slack conversation.

Reply:
This is the bug report about the repos in Full media installation:
https://bugzilla.suse.com/show_bug.cgi?id=1243064
And a question about how to change this PR: to add a new test_data without repos section for media installation?

Reply:
yes, something similar to SP7:
https://openqa.suse.de/tests/17638591#step/validate_repos/3